### PR TITLE
Add plugin helper for nginx plugin

### PR DESCRIPTION
**Pull request by alexdias**
So it knows how to read an nginx's log if nginx is running on a
container.

### CODE REVIEW

File: outlyer/plugin_helper/container/_nginx.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import inspect
+import os
+import tempfile
+from outlyer.plugin_helper import container
+
+def reverse_read(fname, separator=os.linesep):
```
Comment (aymanosman) [inline outlyer/plugin_helper/container/_nginx.py]: what's this doing here when it's duplicated in the nginx.py plugin?

**Review by aymanosman — APPROVED** [review]

**Comment by tradel** [comment]
Can we put the import into a `try/catch` block so that it doesn't fail if people are running an older version of the agent?

**Review by tradel — CHANGES_REQUESTED** [review]
Add `try/catch` block around `plugin_helper` import.

**Comment by alexdias** [comment]
@tradel Sure, but which import do you mean? This one `from outlyer.plugin_helper import container`?

**Comment by alexdias** [comment]
@tradel nevermind just saw the update here :)

**Comment by alexdias** [comment]
@tradel How do older versions of the agent handle this? In order words, if the import fails, should we do anything or just return?

**Comment by tradel** [comment]
You're right, I commented on the wrong PR. Never mind me, carry on.

**Review by tradel — APPROVED** [review]

**Pull request merged**